addon/services/url.js

```diff
@@ -0,0 +1,18 @@
+import Service from '@ember/service';
+import { getOwner } from '@ember/application';
+
+export default class URLService extends Service {
```

Comment:
I'm actually a bit surprised to see a URL service 🤔  we already have a url-helper that was removed from empress-blog and into a "ghost helpers" component https://github.com/empress/empress-blog-ghost-helpers/blob/master/addon/helpers/url.js

Do you know if they do the same thing? if so then maybe we should deprecate the url **service** in favour of the helper 🤔  what do you think?